DOCUMENTATION.md

```diff
@@ -564,6 +565,31 @@ const client = await Client.newHazelcastClient(cfg);
+| hazelcast.client.socket.no.delay                       | true                                | boolean         | The [setNoDelay](https://nodejs.org/api/net.html#net_socket_setnodelay_nodelay) setting for connections.                                                                                                                                                                                                                                                                                                            |
+| hazelcast.client.statistics.enabled                    | false                               | boolean         | If set to `true`, it enables collecting the client statistics and sending them to the cluster. When it is `true` you can monitor the clients that are connected to your Hazelcast cluster, using Hazelcast Management Center.                                                                                                                                                                                       |
+| hazelcast.client.statistics.period.seconds             | 3                                   | number          | Period in seconds the client statistics are collected and sent to the cluster.                                                                                                                                                                                                                                                                                                                                      |
+| hazelcast.discovery.public.ip.enabled                  | null(detection enabled)             | boolean or null | When set to `true`, the client will assume that it needs to use public IP addresses reported by members. When set to `false`, the client will always use private addresses reported by members.                                                                                                                                                                                                                     |
+| hazelcast.invalidation.max.tolerated.miss.count        | 10                                  | number          | If missed invalidation count is bigger than this value, relevant cached data will be made unreachable.                                                                                                                                                                                                                                                                                                              |
```

Comment:
```suggestion
| hazelcast.invalidation.max.tolerated.miss.count        | 10                                  | number          | If missed invalidation count is greater than this value, relevant cached data will be made unreachable.                                                                                                                                                                                                                                                                                                              |
```
Maybe we can put the word `Near cache` somewhere, as this is related to that.

Reply:
Ok I will try to add near cache word. I think `bigger` is ok btw.